uber/config.py

```diff
@@ -10,6 +10,12 @@ def mixin(cls, klass):
                 setattr(cls, attr, getattr(klass, attr))
         return cls
 
```

Comment:
We talked about this today.  Plugins can now call this method to make their own config part of ``c``, but that's totally optional.